src/amp/server/render.tsx

```diff
@@ -72,6 +72,7 @@ export const render = ({ body }: express.Request, res: express.Response) => {
 		if (e instanceof TypeError) {
```

Comment:
not sure if this is the best way of determining the difference between `400` and `500` error?